core/dovecot/conf/ham.script

```diff
@@ -6,6 +6,6 @@ then
 	exit 1
 fi
 
-tee >(rspamc -h $RSPAMD_HOST -P mailu learn_ham /dev/stdin) \
-	>(rspamc -h $RSPAMD_HOST -P mailu -f 11 fuzzy_del /dev/stdin) \
-	| rspamc -h $RSPAMD_HOST -P mailu -f 13 fuzzy_add /dev/stdin
+exec tee >(rspamc -t 3 -h $RSPAMD_HOST -P mailu learn_ham /dev/stdin||true) \
+       >(rspamc -t 3 -h $RSPAMD_HOST -P mailu -f 11 fuzzy_del /dev/stdin||true) \
+       >(rspamc -t 3 -h $RSPAMD_HOST -P mailu -f 13 fuzzy_add /dev/stdin||true)
```

Comment:
Well we want the output, no?
What we don't want is tee to bail out if the return code isn't zero

Reply:
No, the goal was to avoid creating a new process / ensure it gets managed/killed by dovecot. I don't think stdout goes anywhere anyway.

I've made the requested change on ab3dd11d